### ISSUE TITLE
Insufficient contrast on highlighted suggestion

### ISSUE THREAD

**Issue by techbridgedev** [easyops-cn/docusaurus-search-local#425]
The contrast between the background color and text on the highlighted result in the suggestions dropdown is insufficient and fails WCAG recommendations for accessibility.

It's poor in light mode, and unreadable in dark mode.

Please update the module stylesheet to use higher contrast colors for the highlighted result.

**Comment by weareoutman** [comment]
You can customize the style as you want, see https://github.com/easyops-cn/docusaurus-search-local?tab=readme-ov-file#custom-styles

**Comment by techbridgedev** [comment]
Thanks; I didn't see that. But, please consider shipping with default colors that meet accessibility requirements.

**Comment by weareoutman** [comment]
What's your actual color setting of `--ifm-color-primary`? I think maybe you have just set a value to cause the contrast issue.

**Comment by seezee** [comment]
You are correct that I don't use the default value for `--ifm-color-primary`; however, I tested the site without my custom value and it still failed the contrast requirements for normal text.

Here are screenshots showing the contrast with the default values:

<img width="582" alt="Screenshot 2024-06-25 at 3 21 45 PM" src="https://github.com/easyops-cn/docusaurus-search-local/assets/287836/fdb70f42-d1fa-4344-bd2b-39aa7eecc132">
<img width="568" alt="Screenshot 2024-06-25 at 3 21 58 PM" src="https://github.com/easyops-cn/docusaurus-search-local/assets/287836/e11fc95a-3fc0-423e-a027-399bef84ce74">

The white text on the blue highlight is too low contrast.

**Comment by seezee** [comment]
<img width="764" alt="Screenshot 2024-06-25 at 3 28 59 PM" src="https://github.com/easyops-cn/docusaurus-search-local/assets/287836/9ac0e925-a641-41ad-8e11-fa4ba115c8e7">
